### PR TITLE
Expose trigger_deployment rake task

### PR DESCRIPTION
**Pull request by RKushnir**

### CODE REVIEW

**Comment by RKushnir** [comment]
Sorry, not ready yet

**Comment by RKushnir** [comment]
Now ready

File: lib/tasks/deploy.rake [hunk 1]
```diff
@@ -48,10 +48,22 @@ namespace :deploy do
...
 
-    trigger_deployment(application, build, branch) if branch == 'staging'
+    trigger_deployment(application, build, branch)
+  end
+
+  namespace :docker do
+    desc 'triggers deployment of an existing docker container'
+    task trigger_deployment: [:environment], %i(application) do |_, args| do
+      application = args[:application]
+      build = ENV['CIRCLE_BUILD_NUM']
```
Comment (dziemba) [inline lib/tasks/deploy.rake:58]: Now this is duplicated. Can we simplify it?

Comment (RKushnir) [inline lib/tasks/deploy.rake:58]: You mean, pass these as parameters to the rake task?

```
rake trigger[application,$CIRCLE_BUILD_NUM,$CIRCLE_BRANCH]
```

Comment (dziemba) [inline lib/tasks/deploy.rake:58]: Maybe just change the existing task?

``` sh
rake docker:deploy # behavior as before
rake docker:deploy TARGETS=backoffice,nevaly-bo # deploys to the given targets
```

**Comment by RKushnir** [comment]
https://github.com/ad2games/docker-deploy/pull/4

**Pull request closed without merging**